isis/tests/FunctionalTestsTgocassisstitch.cpp

```diff
@@ -22,12 +22,14 @@ TEST(TgoCassisstitch, TgoCassisstitchMultiframeTest) {
   cubeList->append("data/tgoCassis/tgocassisstitch/CAS-MCO-2016-11-22T16.16.16.833-RED-01006-B1_crop.cub");
   cubeList->append("data/tgoCassis/tgocassisstitch/CAS-MCO-2016-11-22T16.16.16.833-NIR-02006-B1_crop.cub");
   cubeList->append("data/tgoCassis/tgocassisstitch/CAS-MCO-2016-11-22T16.16.16.833-PAN-00006-B1_crop.cub");
-
+  
   QString cubeListFile = prefix.path() + "/cubelist.lis";
   cubeList->write(cubeListFile);
 
+  QString cubeName = "default";
+
   QVector<QString> args = {"fromlist=" + cubeListFile,
-                           "outputprefix=" + prefix.path() + "/CAS-MCO"};
+                           "outputprefix=" + prefix.path() + "/CAS-MCO-"};
```

Comment:
One last thing, it's concerning that we are having to add dashes in manually for tests. We shouldn't need to change in the input to get the same output.

It seems like we should be placing a dash between the prefix and the cube name in the `frameFileName` in the tgocassisstitch app

Reply:
Addressed with my latest PR and I've just checked that tests were passing for me locally @acpaquette